ray-operator/config/crd/bases/ray.io_rayclusters.yaml

```diff
@@ -34,6 +34,51 @@ spec:
           spec:
             description: Specification of the desired behavior of the RayCluster.
             properties:
+              autoscalerOptions:
+                description: AutoscalerOptions specifies optional configuration for
+                  the Ray autoscaler.
+                properties:
+                  idleTimeoutSeconds:
+                    description: IdleTimeoutSeconds is the number of seconds to wait
+                      before scaling down a worker pod which is not us
+                    format: int32
+                    type: integer
+                  image:
+                    description: Image optionally overrides the autoscaler's container
+                      image.
+                    type: string
+                  resources:
+                    description: Resources specifies resource requests and limits
+                      for the autoscaler container.
+                    properties:
+                      limits:
+                        additionalProperties:
+                          anyOf:
+                          - type: integer
+                          - type: string
+                          pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                          x-kubernetes-int-or-string: true
+                        description: 'Limits describes the maximum amount of compute
+                          resources allowed. More info: https://kubernetes.'
+                        type: object
+                      requests:
+                        additionalProperties:
+                          anyOf:
+                          - type: integer
+                          - type: string
+                          pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                          x-kubernetes-int-or-string: true
+                        description: Requests describes the minimum amount of compute
+                          resources required.
+                        type: object
+                    type: object
+                  upscalingMode:
+                    description: UpscalineMode is "Default" or "Aggressive.
+                    enum:
+                    - Default
+                    - Aggressive
```

Comment:
I am just wondering, where is the `upscaling_speed`?

Reply:
The allowed values of "Default" and "Aggressive" will translate into an `upscaling_speed` of 1 and 1000, respectively.
(1000 is a stand-in for infinity.)

The reason is that we've never seen anyone have a need for anything strictly between 1 and infinity.

An upcoming Ray PR will have the autoscaler parse the new field.